ci/win_test_installer.cmd

```diff
@@ -22,7 +22,7 @@ C:\Python39\python -c "import neuron; neuron.test(); quit()" || set "errorfound=
 C:\Python310\python -c "import neuron; neuron.test(); quit()" || set "errorfound=y"
 C:\Python311\python -c "import neuron; neuron.test(); quit()" || set "errorfound=y"
 :: install numpy dependency
-python -m pip install "numpy<2"
+python -m pip install "numpy"
```

Comment:
Every manual installation here makes me wonder: did we just do this to get the version right because something else installed a newer Numpy?

In that case, the entire line should not be needed any longer.

If it is needed, why is it not present for all other Python versions above?

Reply:
Fair point, I just grepped for the pins quickly without looking much. Let me try it without.